packages/fetch/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export { formatApiError, formatApiResponse } from './format'
+export { schemaModel } from './schemaModel'
+export { request } from './request'
+export type {
+  ApiError as APIError,
+  ApiResponse as APIResponse,
```

Comment:
I don't think we need to alias here. We can do a find/replace in scalar-org

Reply:
I agree. This is an interesting functionality of symbol renaming. It seems to automatically alias exports if the name changes. I removed the alias'